src/Tests/Creation/When_creating_queues.cs

```diff
@@ -166,23 +166,19 @@ public class When_creating_queues
         }
 
         [Test]
-        public async Task Should_set_MaxDeliveryCount_to_specified_value_for_immediate_retries_not_enabled()
+        public async Task Should_set_MaxDeliveryCount_to_1_for_disabled_immediate_retries()
         {
             var namespaceManager = new NamespaceManagerAdapterInternal(NamespaceManager.CreateFromConnectionString(AzureServiceBusConnectionString.Value));
 
             var settings = DefaultConfigurationValues.Apply(SettingsHolderFactory.BuildWithSerializer());
 
-            var extensions = new TransportExtensions<AzureServiceBusTransport>(settings);
-
-            extensions.Queues().MaxDeliveryCount(100);
-
             var creator = new AzureServiceBusQueueCreator(settings.Get<TopologySettings>().QueueSettings, settings);
 
             await creator.Create("myqueue", namespaceManager);
 
             var real = await namespaceManager.GetQueue("myqueue");
 
-            Assert.AreEqual(100, real.MaxDeliveryCount);
+            Assert.AreEqual(1, real.MaxDeliveryCount);
```

Comment:
So this test relies on a default value? Since the core controls that setting could we at least add a hardcoded value under the settings key?

Reply:
> could we at least add a hardcoded value under the settings key?

Yes. 
Curious what you meant by "at least". What would _more_ look like?

Reply:
Why would it be a settings key rather than a constant? Settings key is the "key" used to locate a specific piece of info under settings. In this case it's a plain constant. Will introduce a constant to remove a hard-coded value.

Reply:
Sorry my explanation was not clear enough. The previous test set it to 100 and then asserted the count to be 100. Now we are no longer setting a value. Now when you read the test you might be wondering where the value is coming from. A constant would solve it or setting an explicit value